fv3core/stencils/d_sw.py

```diff
@@ -331,9 +348,8 @@ def heat_source_from_vorticity_damping(
         heat_source[0, 0, 0] = delp * (
             heat_source - 0.25 * kinetic_energy_fraction_to_damp * dampterm
         )
-        dissipation_estimate[0, 0, 0] = (
-            -dampterm if calculate_dissipation_estimate == 1 else dissipation_estimate
-        )
+        if __INLINED(namelist.do_skeb == 1):
+            dissipation_estimate[0, 0, 0] = -dampterm
```

Comment:
why 0's on the lhs?

Reply:
there is a mix of this all over the code base, we still all haven't agreed on what is preferred :). I actually prefer no 0's, this is just moved code